phys/module_mp_nssl_2mom.F

```diff
@@ -2241,12 +2243,17 @@ SUBROUTINE nssl_2mom_driver(qv, qc, qr, qi, qs, qh, qhl, ccw, crw, cci, csw, chw
    ! calc dbz
       
       IF ( .true. ) THEN
+      IF ( present(ke_diag) ) THEN
```

Comment:
@davegill @MicroTed I doubt this is the right check: ke_diag is always there, always defined. Perhaps ke_diag can be passed into the radar routine directly.

Reply:
@weiwangncar Yes, going forward for WRF it is always there, but I also back-port current code to older versions we're still using around here (and 3 other models) where it doesn't exist.  I appreciate the thought!

That "IF ( .true.) " is old debugging cruft, though... hahaha.